app/scripts/views/complete_reset_password.js

```diff
@@ -121,7 +121,7 @@ function (Cocktail, BaseView, FormView, Template, PasswordMixin,
           return self.fxaClient.signIn(email, password, self.relier);
         }).then(function (accountData) {
           var account = self.user.initAccount(accountData);
-          self._interTabChannel.emit('login', accountData);
+          self._interTabChannel.send('login', accountData);
```

Comment:
I just wanted to make the interface for `interTabChannel` consistent with the other channels. I'm not sure we'd want to broadcast `accountData` over all the channels.